### ISSUE TITLE
Describe ECS with sidecar agent configuration (stdout/stderr)

### ISSUE THREAD

**Issue by jensenbox** [namespace-ee/django-datadog-logger#14]
* Django DataDog Logger version: latest
* Python version: 3.11
* Operating System: AWS Linux 2

### Description

I am working on a new project that uses Django.

This library looks to do everything we would want to do except there is a mismatch in the way it wants to collect the logs and the prescribed way for containers - specifically AWS ECS.

The package wants the application to write the log files to disk. Writing files while inside a container is generally a very bad thing as it can run out of disk space as well as other issues.

Instead, the prescribed way is to emit the log data as something called "STDOUT" and "STDERR" - those are basically data streams and the Datadog Agent prefers this method.

So basically these two pages disagree with each other:
* https://docs.datadoghq.com/logs/log_collection/python/ - mentions this library
* https://docs.datadoghq.com/containers/amazon_ecs/logs/

The ask I have is if there a way to wire it up in the prescribed STDOUT and STDERR way. Ideally the docs would be updated to follow the prescribed way as well.

**Comment by lnagel** [comment]
This package does not imply logging to disk, it only implements the formatter. We are using this perfectly fine with `logging.StreamHandler` instead of `logging.handlers.WatchedFileHandler`.
